karapace/schema_reader.py

```diff
@@ -306,49 +302,52 @@ def handle_messages(self) -> None:
             if are_we_master is True:
                 watch_offsets = True
 
-        for _, msgs in raw_msgs.items():
-            schema_records_processed_keymode_canonical = 0
-            schema_records_processed_keymode_deprecated_karapace = 0
-            for msg in msgs:
+        schema_records_processed_keymode_canonical = 0
```

Comment:
One level less looping, as the new client doesn't return messages separated by partition.

Reply:
Nice! 👍